src/docs/deployment/cd.md

```diff
@@ -117,7 +117,8 @@ untrusted, you won't be leaving your credentials like your Play Store service
 account JSON or your iTunes distribution certificate on the server.
 
 Continuous Integration (CI) systems generally support encrypted environment 
-variables to store private data.
+variables to store private data. You can pass these environment variables 
+using `--dart-define MY_VAR=MY_VALUE` while building the app.
```

Comment:
Can we link to further documentation on this? I'm curious what I can pass `--dart-define` to.

Reply:
Yeah, this is a deficiency of our docs! `dart-define` is a way to pass compile-time variables to the `flutter run` or `flutter build` commands. I gather that you can even use it to pass values down to the native level. Please file a bug about this omission on the flutter.dev repo. In the short term, maybe provide a full example *or* link to the only page on the site where dart-define is used, [Report errors to a service](https://flutter.dev/docs/cookbook/maintenance/error-reporting).

Reply:
Raised https://github.com/flutter/website/issues/5853 =)